src/main/scala/ztis/wykop/WykopAPI.scala

```diff
@@ -38,4 +45,52 @@ class WykopAPI(config: Config) extends StrictLogging {
 
     entries
   }
+
+  def usersFollowingUser(userName: String): Vector[String] = {
+    var page = 1
+    var someFollowersLeft = true
+    val buffer = ArrayBuffer.empty[String]
+
+    while (someFollowersLeft) {
+      val oldSize = buffer.size
+      val url = usersFollowingUserURL(userName, page)
+      val digest = calculateDigest(url)
+
+      downloadUsersFollowingUser(url, digest, buffer)
+      val newSize = buffer.size
+
+      if (oldSize == newSize) {
+        someFollowersLeft = false
+      } else {
+        page += 1
+      }
+    }
+
+    buffer.toVector
+  }
```

Comment:
But less performant - you would have unnecessary object allocation for each collection. For now this method is used only in this place so it can be tailored to its needs.